src/getdns/getdns.h.in

```diff
@@ -46,7 +46,10 @@ extern "C" {
 
 #define GETDNS_COMPILATION_COMMENT "@GETDNS_COMPILATION_COMMENT@"
 
-/* Return values */
+/**
+ * \defgroup returntypes Return types
```

Comment:
Op 22-06-16 om 14:13 schreef saradickinson:
> In src/getdns/getdns.h.in
> <https://github.com/getdnsapi/getdns/pull/186#discussion_r68041693>:
> 
>> @@ -46,7 +46,10 @@ extern "C" {
>>  
>>  #define GETDNS_COMPILATION_COMMENT "@GETDNS_COMPILATION_COMMENT@"
>>  
>> -/* Return values */
>> +/**
>> + * \defgroup returntypes Return types
> 
> On 22 Jun 2016, at 12:48, wtoorop ***@***.***> wrote: In
> src/getdns/getdns.h.in <#186 (comment)
> <https://github.com/getdnsapi/getdns/pull/186#discussion_r68038731>>: >
> @@ -46,7 +46,10 @@ extern "C" { > > #define GETDNS_COMPILATION_COMMENT
> "@GETDNS_COMPILATION_COMMENT@" > > -/* Return values */ > +/** > + *
> \defgroup returntypes Return types Return types? Not return values?

> Hi Willem, What I did was for consistency was call all those constants
> that are defined within a type (via typedef enum {}) as a ‘Type' and
> those that are straight #defines ‘Values'.

That distinction is confusing to me.  Constants are always values (in
C), whether they come from #defines, enums or from static declarations
(i.e. static const int four = 4; ).

I believe that the usage of the word "type" when referring to those
values before, was semantic and not referring to a property of the C
language.
I.e. "Namespace types" meaning "Kinds of namespaces"
     "Resolution types" meaning "Ways to do resolution."
etc.

Does that make sense?


> This maybe too implementation
> specific, but the previous naming seemed a bit arbitrary to me. I don’t
> mind too much what we decide but I do like consistency…. Sara.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/getdnsapi/getdns/pull/186/files/ebad77b0e3be73d7bd8390eee4fe029c8d027d3f#r68041693>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE-Rkq0RjkjLzMl-gDtXeTRDIRfbNTXIks5qOScAgaJpZM4I4frN>.
>